tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -109,6 +117,13 @@ When('the user renames file/folder {string} to an invalid name {string} using th
   return client.page.FilesPageElement.filesList().renameFile(fromName, toName, false)
 })
 
+When('the user renames the following files using the webUI', function (table) {
+  for (const line of table.rows()) {
+    client.page.FilesPageElement.filesList().renameFile(line[0], line[1], false)
```

Comment:
```suggestion
    client.page.FilesPageElement.filesList().renameFile(line[0], line[1])
```
I think this step is expected to succed?